app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java

```diff
@@ -1,13 +1,124 @@
 package com.greenfox.opal.gitinder;
 
+import android.content.DialogInterface;
+import android.os.Handler;
+import android.support.v7.app.AlertDialog;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.util.Log;
+import com.google.api.client.auth.oauth2.BearerToken;
+import com.google.api.client.auth.oauth2.ClientParametersAuthentication;
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.extensions.android.http.AndroidHttp;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.wuman.android.auth.AuthorizationDialogController;
+import com.wuman.android.auth.AuthorizationFlow;
+import com.wuman.android.auth.DialogFragmentController;
+import com.wuman.android.auth.OAuthManager;
+import com.wuman.android.auth.OAuthManager.OAuthCallback;
+import com.wuman.android.auth.OAuthManager.OAuthFuture;
+import java.io.IOException;
 
 public class LoginActivity extends AppCompatActivity {
 
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_login);
+
+    new Handler().post(new Runnable() {
+      @Override
+      public void run() {
+        loginAlert();
+      }
+    });
+  }
```

Comment:
@balintvecsey handler stuff is not needed if you put this into the `onResume` callback method instead of `onCreate`, also the creation of the `AlertDialog` can be in the method, no need for a method just for that